docs/scenarios_authoring/Enrichers.md

```diff
@@ -41,6 +41,28 @@ DatabaseLookupEnricher is a specialized look-up component; it returns all column
 
 ![alt_text](img/databaseLookupEnricher.png "databaseLookup Enricher")
 
+### DecisionTable enricher
+
+This component allows the user to match input to a pre-defined result. Please be wary that this component works a bit different than a classical decision table([Decision tables in Wikipedia](https://en.wikipedia.org/wiki/Decision_table)). It consists of the fields:
+- Name: the name of the component.
+- Decision Table: a user defined table of type TabularTypedData, discussed below.
+- Match condition: a Boolean SpEL expression that matches input to table rows, discussed below.
+- Output: name of the output variable which will hold data generated by the component.
```

Comment:
let's write explicitly more about the "data generated". What's the type of the data? What does the data hold?